modules/microshift-embed-microshift-build-image.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-offline-deploy.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-microshift-build-image-offline-deployment_{context}"]
+= Building the rpm-ostree image for offline deployments
+
+You can use Image Builder to create `rpm-ostree` system images with embedded {microshift-short} container images. To embed container images, you must add the image references to your Image Builder blueprint.
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have created and updated an {op-system-ostree} image blueprint for offline use.
+* You have updated the `/etc/osbuild-worker/osbuild-worker.toml` configuration file for offline use.
+
+.Procedure
+
+. Import the modified blueprint TOML file into Image Builder by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints push <my_blueprint_name>
+----
+Replace `<my_blueprint_name>` with the name of your TOML blueprint file.
+
+. List existing blueprints to verify that the created blueprint is successfully pushed and exists by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints show blueprint-name
+----
+
+. Build the `rpm-ostree` commit with the embedded container image by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli compose start
```

Comment:
This command is generating an error

' sudo composer-cli compose start
Error: Invalid number of arguments
Usage:
  composer-cli compose start BLUEPRINT TYPE [IMAGE-NAME PROFILE.TOML] [flags]

Examples:
  composer-cli compose start tmux-image qcow2
  composer-cli compose start tmux-image qcow2 --size 4096
  composer-cli compose start tmux-image ami ami-name aws-upload.toml

Flags:
  -h, --help        help for start
      --size uint   Size of image in MiB

Global Flags:
  -a, --api int         Server API Version to use (default 1)
  -j, --json            Output the raw JSON response instead of the normal output
      --log string      Path to optional logfile
  -s, --socket string   Path to the server's socket file (default "/run/weldr/api.socket")
      --test int        Pass test mode to compose. 1=Mock compose with fail. 2=Mock compose with finished.
      --timeout int     Timeout to use for server communication. Set to 0 for no timeout (default 240)'

Reply:
@dhensel-rh Try the following syntax per the RHEL docs: 
$ sudo composer-cli compose start <my_blueprint_name> edge-commit

Reply:
Sure, we should fix the command to be of correct syntax, or remove it all together and refer to the section where we have the procedure documented in full. I would prefer the latter.